Chore(github): add pr template (#59)
fastify/website-next: .github/PULL_REQUEST_TEMPLATE.md.MD

## Description

<!-- Write a brief description of the changes introduced by this PR -->

## Related Issues

<!--
  Link to the issue that is fixed by this PR (if there is one)
  e.g. Fixes #1234, Addresses #1234, Related to #1234, etc.
-->

> ### Check List

Reviewer: ```suggestion
## Check List
```

<!--
ATTENTION

> Please follow this check list to ensure that you've followed all items before opening this PR

Reviewer: ```suggestion
Please follow this check list to ensure you've followed all items before opening this PR.
A PR will be merged only when the CI pipeline is successful and the approval process is completed.
```

-->

> - [ ] I have read the [Contributing Guidelines]() and made commit messages that follow the guideline.
> - [ ] I have run `npm run format` to format my code.

Reviewer: No need to add the git comment thing, we squash&merge so it doesn't matter

```suggestion
- [ ] I have read the [Contributing Guidelines](./CONTRIBUTING.md)
```
